docs/fidesops/docs/guides/manual_data.md

```diff
@@ -7,17 +7,16 @@ In this section we'll cover:
 
 ## Overview
 
-Not all data can be automatically retrieved: some data will need to be manually provided by you.
-Similar to how you define [datasets](datasets.md) to annotate your owned databases and your third-party
-saas integrations, you can define a dataset to describe the types of manual fields you plan to upload
-and any dependencies between these manual collections and other collections.
+Not all data can be automatically retrieved. When services have no external API, or when user data is held in a physical location, you can define a [dataset](datasets.md) to describe the types of manual fields you plan to upload, as well as any dependencies between these manual collections and other collections. 
+
+When a manual dataset is defined, an in-progress access request will pause until the data is added manually, and then resume execution.
 
 ## Describing a manual dataset
 
-In this example, we have a manual dataset with one `storage_unit` collection.  `email` is 
-an identity, so this is the field needed to locate the  `box_id` in the storage unit.
+In the following example, the Manual Dataset contains one `storage_unit` collection.  `email` is 
+defined as the unit's [identity](datasets.md#field-members), which will then be used to retrieve the `box_id` in the storage unit.
 
-You would need to add both a Manual [ConnectionConfig](database_connectors.md) and this Manual Dataset to that ConnectionConfig.
+To add a Manual Dataset, first create a [Manual ConnectionConfig](database_connectors.md#example-6-manual-connectionconfig). The following Manual Dataset can then be added to the new ConnectionConfig:
```

Comment:
Hm, I didn't go into detail here because this part isn't new behavior: every ConnectionConfig can have multiple DatasetConfigs attached to them, I've just added a new type of ConnectionConfig.

Looking further, I thought this was covered in the guides, but I don't see it.  Is this detail you could add @conceptualshark?  I would think it would be an extension to `datasets.md`.

Generally you create a ConnectionConfig which describes how to connect to postgres, mongodb, etc.  and then you create a DatasetConfig which describes the data contained there.  This is not new behavior added here, this is something you'd do for every connector type.  We describe what should be in the datasets here: https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/guides/datasets.md, but I don't see further explanation of how to actually add that to fidesops with an API request.

Reply:
Thanks for the clarification! I checked through the datasets doc and wasn't sure if I was missing something - a whole step makes sense. I can take a look and try to write something up for that piece. 👍

Reply:
cool thank you! it's a PATCH to `/api/v1/connection/connection_key/dataset`.  Until recently you had to convert the dataset yaml file into JSON to send in the request body, but I think with @nschoenbrot's work you can now send in the yaml directly